### ISSUE TITLE
burp2-server provides burp-server

### ISSUE THREAD

**Issue by kronos-pbrideau** [yopito/fedora-epel-pkg#2]
I've notice what appear to be an inconsistency, but i'm not sure if this is intended or a real inconsistency...

Client package provides `burp2-client`, but server package provides `burp-server` instead of `burp2-server`

Consequence is i try to remove old burp-server as it conflicts:

```
$ yum install burp2-client burp2-server
Loaded plugins: priorities, versionlock
267 packages excluded due to repository priority protections
Resolving Dependencies
--> Running transaction check
---> Package burp2-client.x86_64 0:2.0.44-1.el7.centos will be installed
---> Package burp2-server.x86_64 0:2.0.44-1.el7.centos will be installed
--> Processing Conflict: burp2-client-2.0.44-1.el7.centos.x86_64 conflicts burp-client
--> Processing Conflict: burp2-client-2.0.44-1.el7.centos.x86_64 conflicts burp < 2.0
--> Finished Dependency Resolution
Error: burp2-client conflicts with burp-client-1.4.40-1.el7.x86_64
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

So ok, we must remove burp-(server|client) prior to install burp2-(server|client), no problems!

But when automating with puppet, there is no difference between burp2-server and burp-server, so it always uninstall burp2-server with this syntax

```
package { 'burp-server' :
  ensure => absent,
} ->
package { 'burp2-server' :
  ensure => present
}
```

Because when burp2-server is installed, notice how the system want to uninstall burp2-server even when i want only the old version to be uninstalled:

```
$ yum remove burp-server -q

======================================================================================================================
 Package                     Arch                  Version                               Repository              Size
======================================================================================================================
Removing:
 burp2-server                x86_64                2.0.44-1.el7.centos                   @yopito                 32 k

Transaction Summary
======================================================================================================================
Remove  1 Package

Is this ok [y/N]: 
```

```
%package client
Summary:    burp backup client
Group:      Backup Server
Requires:   librsync >= 1.0
Provides:   burp = %{version}-%{release}
Provides:   burp2-client = %{version}-%{release}
```

```
%package server
Summary:    burp backup server
Group:      Backup Server
Requires:   burp2-client%{?_isa} = %{version}-%{release}
Requires:   openssl-perl
Provides:   burp-server = %{version}-%{release}
Provides:   bedup = %{version}-%{release}
Provides:   vss_strip = %{version}-%{release}
```

TLDR: Changing the package to `Provides: burp2-server` instead of `Provides: burp-server` would solve this problem for me, what do you think about this?

**Comment by yopito** [comment]
hello,

thanks for pointing this. I'll review it quietly ASAP.
This is probably a left quirk: at the beginning, the `burp` package from EPEL was providing both -server and -client. So in order to avoid people get confused or doing wrong, we started with the EPEL guy to explicitely break the upgrade, in order to let them aware of the new packaging split.
stay tuned.

**Comment by yopito** [comment]
Hello,

Sorry for the delay. I plan to remove  "`Provides: burp-server`" in `burp2-server` package, since it seems obviously uneeded. Then "burp2-server" (as a requirement) will be provided by the package's name, that is supposed to be enough.

Have to test various upgrade scenarios first before change it in my public packages, stay tuned.

FYI, I've packaged burp2 with "don't make big changes automatically" in mind:
- prevent an automatic upgrade from 1.x to 2.x, since they are quite different (and burp 1.x was the "stable" release: people have to (explicitely) uninstall 1.x first if 2.x is wanted
- also prevent automatic switch from "monotithic" burp package of others that is usually named "burp" (EPEL burp < 1.4.40 for instance) for the same reasons than previous point.

**Comment by yopito** [comment]
Fedora COPR infrastructure is currently building burp2 2.0.44-2 packages. Should be available soon once propagated onto COPR yum mirrors.
